tests/platform_tests/mellanox/test_check_sfp_presence.py

```diff
@@ -28,7 +27,7 @@ def test_check_sfp_presence(duthost, conn_graph_facts):
         assert intf in presence_list, "Wrong interface name in the output %s" % str(presence_list)
         assert 'Present' in presence_list, "Status is not expected, output %s" % str(presence_list)
 
-        check_sysfs_output = duthost.command(check_qsfp_sysfs_command.format(str(sfp_id)))
+        check_sysfs_output = duthost.command(sysfs_path.format(str(sfp_id)))
```

Comment:
I suggest completely remove this SFP sysfs check since in the future SFP sysfs will be deprecated, we don't want to change it again.

Reply:
@shlomibitton Do you like to address this review comments in this PR or in separate PR?

Reply:
@keboliu 
I have removed the sysfs check from this test case.
It is a cross check between two presence commands, sysfs is deprecated thus removed.
@wangxin it is addressed here